patches/core/ungoogled-chromium/disable-google-host-detection.patch

```diff
@@ -578,3 +539,14 @@
 -  }
  }
  #endif
+--- a/components/page_load_metrics/common/page_load_metrics_util.cc
++++ b/components/page_load_metrics/common/page_load_metrics_util.cc
+@@ -47,7 +47,7 @@ base::Optional<std::string> GetGoogleHos
+ }
+ 
+ bool IsGoogleHostname(const GURL& url) {
+-  return GetGoogleHostnamePrefix(url).has_value();
```

Comment:
Is `GetGoogleHostnamePrefix` use anywhere else? I noticed you remove the portion that stubbed out `GetGoogleHostnamePrefix` and just removed its invocation here.

Reply:
How strange. I do not recall changing the patch in this way. I will have to fix that.